lib/spandex_ecto/ecto_logger.ex

```diff
@@ -3,13 +3,16 @@ defmodule SpandexEcto.EctoLogger do
   A trace builder that can be given to ecto as a logger. It will try to get
   the trace_id and span_id from the caller pid in the case that the particular
   query is being run asynchronously (as in the case of parallel preloads).
-  """
 
-  alias Spandex.{
-    Span,
-    SpanContext,
-    Trace
-  }
+  This module used to attempt to continue a trace in the case of parallel preloads.
```

Comment:
I don't think we need this not lingering in the module docs forever, given that it's in the README too.